perseo-main/src/main/java/com/telefonica/iot/perseo/RulesServlet.java

```diff
@@ -134,16 +143,22 @@ protected void doPut(HttpServletRequest request, HttpServletResponse response)
         Utils.putCorrelatorAndTrans(request);
         response.setCharacterEncoding("UTF-8");
         response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        String body = Utils.getBodyAsString(request);
+        try {
+        	
+            String body = Utils.getBodyAsString(request);
+
+            // Save timed rules, which are not activated by events external to the core
+            TimeRulesStore.getInstance().saveTimeRules(body);
+
+            Result r = RulesManager.updateAll(epService, body);
+            response.setStatus(r.getStatusCode());
+            response.getOutputStream().print(Encode.forHtmlContent(r.getMessage()));           
+        } catch (Exception je) {        				
+			logger.error(String.format("error: %s" ,je));
+			response.setStatus(HttpServletResponse.SC_BAD_REQUEST);        	   
```

Comment:
Wrong indent

Reply:
Fixed in 61b8a4c9c7fe92f547917fd56b451a77f39db5f5